ClosedXML/Excel/XLWorkbook_Load.cs

```diff
@@ -1956,47 +1953,47 @@ private static void LoadAutoFilterColumns(AutoFilter af, XLAutoFilter autoFilter
 
                         if (xlDateGroupFilter.DateTimeGrouping >= XLDateTimeGrouping.Year)
                         {
-                            if (dateGroupItem?.Year?.HasValue ?? false)
+                            if (dateGroupItem.Year?.HasValue ?? false)
```

Comment:
You don't have to change this one, but again, including the `?` is my personal preference to future-proof the code.